README.md

```diff
@@ -84,6 +84,9 @@ $ make test
 ```
 
 2. Run the e2e tests:
+
+> *NOTE*: Any changes made to content in the `langs` directory causes the command to fail halfway through the tests.
```

Comment:
I think a better wording would be something like:

> Note: Any changes to the languages need to be re-built with `build-langs` before e2e will pick up the changes.

Reply:
Oh boy, does that mean my issue with Civet and `NODE_ICU_DATA` would not have occurred if I'd just built my image with `./build-langs --no-push Civet`? I thought that was _your_ responsibility as administrator since we cannot push hence `--no-push`? Trying to fully understand it, James.

Reply:
It wouldn't have fixed CI but yes you can test any language change locally by building images for your box without pushing.

Reply:
I'll push a rewording later in the evening. Busy times 😅